hometown.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <!-- meta data -->
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Chisinau</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- CSS link -->
+    <link rel="stylesheet" type="text/css" media="screen" href="/assets/css/style.css">
+    <!--    Google Fonts-->
+    <link href="https://fonts.googleapis.com/css?family=PT+Serif" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+</head>
+
+<body>
+    <div class="container">
+        <!-- Header+Navbar -->
+        <header>
+            <nav>
+                <ul>
+                    <li><a href="#his"><img class="ico" src="/assets/icons/landmark-solid.svg"><span>History</span></a></li>
+                    <li><a href="#loca"><img class="ico" src="/assets/icons/search-location-solid.svg"><span>Location</span></a></li>
+                    <li><a href="#imag"><img class="ico" src="/assets/icons/images-regular.svg"><span>Images</span></a></li>
+                    <li><a href="#sigh"><img class="ico" src="/assets/icons/eye-regular.svg"><span>Sights</span></a></li>
+                </ul>
+            </nav>
+        </header>
+        <!-- First Section -->
+        <div class="part1">
+            <div class="frame">
+                <div class="head">
+                    <h1>Chisinau</h1>
+                </div>
+            </div>
+        </div>
+<!-- Section History -->
+        <div class="part2">
```

Comment:
avoid numbers in class names.
either give a generic class that you can use here and there or an id for spesific styling
maybe i should stop doing that as well =)